src/components/search-panel/SearchPanelResults.tsx

```diff
@@ -49,16 +51,31 @@ const SearchPanelResults = (props: SearchPanelResultsProps) => {
 		showToggler,
 		next,
 		prev,
+		formCollapsed,
+		setFormCollapsed,
 	} = props;
 
 	function computeKey(index: number) {
-		const item = resultGroups[index][0] || 'key';
+		const [, results] = resultGroups[index];
+		const item = results[0];
 
 		return isEventNode(item) ? item.eventId : item.messageId;
 	}
 
+	const onScroll = (e: React.UIEvent<HTMLDivElement, UIEvent>) => {
+		if (e.target === e.currentTarget) {
```

Comment:
It's bugged when after collapsing form, results take more space and items take less space then results height, so after onScrollEvent occurs its scrollTop is 0 and its collapsing

Reply:
I removed this functionality temporarily